.travis.yml

```diff
@@ -12,17 +11,11 @@ os:
 
 before_install:
   - |
-    if [ $TRAVIS_OS_NAME == "osx" ]; then
-      brew update
-      brew install openssl
-    fi
+    brew update
+    brew install openssl
   - |
     echo -n | openssl s_client -connect scan.coverity.com:443 | sed -ne '/-BEGIN CERTIFICATE-/,/-END CERTIFICATE-/p' >/tmp/cert.crt
-    if [ $TRAVIS_OS_NAME == "linux" ]; then
-      cat /tmp/cert.crt >> /etc/ssl/certs/ca-certificates.crt
-    elif [ $TRAVIS_OS_NAME == "osx" ]; then
-      sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain /tmp/cert.crt
-    fi
+    sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain /tmp/cert.crt
```

Comment:
Cool! Thanks for this btw. Where can we go to see the results once it's being run through the code scanners?

Reply:
For BSMtrace: https://scan.coverity.com/projects/bsmtrace

For OpenBSM I added a badge to the README today, the link there is https://scan.coverity.com/projects/openbsm

The BSMtrace scan hasn't run yet due to the huge Travis OSX backlog.